docs/design.md

```diff
@@ -11,6 +11,7 @@
     - [Security](#security)
     - [Volume Persistency](#volume-persistency)
     - [Capacity-aware pod scheduling](#capacity-aware-pod-scheduling)
+    - [Operator API](#operator-api)
```

Comment:
Let's move the API part to the "installation and usage" page. It's much too detailed for a high level design description.

What belongs here is the part about the key aspects of the operator:
- that the operator runs inside the cluster
- driver deployment controlled by a cluster-scoped CR
- the operator creates the driver objects, some of the in its own namespace, some of them at cluster scope

Reply:
> Let's move the API part to the "installation and usage" page. It's much too detailed for a high level design description.

API description part of installation steps feels me like not the right place. So, I kept it as a separate section. I believe `Operator API` itself deserves a self-contained section but not sure which is the right place.

Reply:
Yes, API needs to be a separate section at the end of the install.md page.

Reply:
The file itself is a bit misnamed now, but the title "installation and usage" fits because "API" falls under "usage".

Reply:
> Yes, API needs to be a separate section at the end of the install.md page.

Ok, will do that.